src/en/charms-storage.md

```diff
@@ -415,6 +430,72 @@ provider currently supports a single pool configuration attribute:
 
 ### LXD (lxd)
 
+LXD-based models have access to the 'lxd' storage provider. The LXD provider
+does not currently have any specific configuration options.
```

Comment:
Yeah, I fell victim to a copy & paste from the previous sections.

I understand passing storage constraints but not configuration options.

Why expose `driver` when it's integral to `lxd-pool`? IIUC, doing so would allow for incompatible pairs (e.g. btrfs driver with a zfs-based pool)?

Anyway, I don't know how to pass these configuration options. Can you give a few explicit examples, including one where options are passed to LXD?